all.json

```diff
@@ -14,6 +14,48 @@
 		"1inch-network.net",
 		"1inch-wallet.org",
 		"1inch.tk",
+		"accessgains.uk",
+		"acquiomfinances.com",
+		"alpha-aerotek.com",
+		"armetatrade.com",
+		"bittradingenterprises.com",
+		"blue-chip.ltd",
+		"capital-option.co",
+		"capitaltrusts.ltd",
+		"century-trades.com",
```

Comment:
For all these, i.e. the first batch, I honestly cannot evaluate them - they are all trading platforms, I don't see the "standard" steal-your-seed and have not signed up (checked it though), to form see how they try to take advantage.

These are really out of my wheelhouse.

Reply:
If you check tiny details about any of them you will find more details regarding a complex vast network of advance fee scams
the way it works is like this
they spam victims via phone calls and lure them to invest
the portfolio grows on theory, but upon any attempt to withdraw the problems begin.

for example the phone number listed on the first one  `armetatrade.com` is exactly the same as 'itierion.com`

Some of them have fake company IDs, or company ID belonging to other entities.
Others go the extra mile to actually register a firm, but always bogus.

Another giveaway is the source code where you can see fake messages like "person from %your country% just invested xyz dollars"

Third giveaway is copy pasta BS - if you copy paste any text passage you will discover hundreds more.

[All the ones listed here are the same scam](https://twitter.com/dubstard/status/1514837690465890310)

Reply:
another giveaway is 
-social media links are always blank
Investors have same pictures (stock photos - if you Reverse image Search any CEO's picture, you will find 1000 more websites with the same CEO
they are all fake 100%

Reply:
![image](https://user-images.githubusercontent.com/49607867/166194797-5e2d0b90-a956-475a-aba9-31854a3951f8.png)

Reply:
![image](https://user-images.githubusercontent.com/49607867/166195457-5b269ac8-ea17-4223-a9ea-a21a520b2031.png)